includes/Core/Util/Input.php

```diff
@@ -39,6 +59,20 @@ class Input {
 	 *                                      and NULL if the filter fails.
 	 */
 	public function filter( $type, $variable_name, $filter = FILTER_DEFAULT, $options = null ) {
-		return filter_input( $type, $variable_name, $filter, $options );
+		$value = filter_input( $type, $variable_name, $filter, $options );
+
+		// Fallback for environments where filter_input may not work with SERVER or ENV types.
+		if (
+			// Only use this fallback for affected input types.
+			in_array( $type, array( INPUT_SERVER, INPUT_ENV ), true )
```

Comment:
I think we can strip this check and only rely on the `array_key_exists` check below which results in the same.